docs/README.md

```diff
@@ -0,0 +1,114 @@
+
+### Bug Report
+Definition: A bug is a *demonstrable* problem that is caused by the code in the repository.
+
+1. Use the GitHub issue search — check if the issue has already been reported. How to use search function? Check out this [guide](https://help.github.com/articles/using-search-to-filter-issues-and-pull-requests/).
+
+2. Check if the issue has been fixed — try to reproduce it using the latest master or look for closed issues in the current milestone.
+
+3. Isolate the problem — ideally create a reduced test case and a live example.
+
+4. Include a screencast if relevant - Is your issue about a design or front end feature or bug? The most helpful thing in the world is if we can see what you're talking about. Use LICEcap to quickly and easily record a short screencast (24fps) and save it as an animated gif! Embed it directly into your GitHub issue.
+
+5. Include as much info as possible! Use the Bug Report template below or click this link to start creating a bug report with the template automatically.
+
+A good bug report shouldn't leave others needing to chase you up for more information. Be sure to include the details of your environment.
+
+Here is a real example of a great bug report.
+
+
+```sh
+Short and descriptive example bug report title
+
+### Issue Summary
+
+A summary of the issue and the browser/OS environment in which it occurs.
+
+### Steps to Reproduce
+1. This is the first step
+2. This is the second step, etc.
+
+Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?
+
+### Technical details:
+* Operation System: macOS 10.13.4
+* Node Version: 4.4.7
+* Dispatch-proxy Version: master (latest commit: a761de2079dca4df49567b1bddac492f25033985)
+* Internet connections and local IP adresses
+* Browser: Chrome 48.0.2564.109 on Mac OS X 10.10.4
+```
+
+### Feature Request
+If you have ideas, feel free to open an issue and tell us all about it!
+
+Request template:
+```sh
+Short and descriptive example feature request title
+
+### Summary
+
+A summary of the your idea and how it would work.
+
+### Use case model
+As... (a game streamer)
+I want... (make use of two internet connections, both at 100 Mbps upload)
+Because... (I need at least 120 Mbps to stream in 4k at 144 fps)
+```
+
+### Change Request
+Change requests cover both architectural and functional changes to how Ghost works. If you have an idea for a new or different dependency, a refactor, or an improvement to a feature, etc - please be sure to:
+
+1. Use the GitHub search and check someone else didn't get there first
+
+2. Take a moment to think about the best way to make a case for, and explain what you're thinking as it's up to you to convince the project's leaders the change is worthwhile. Some questions to consider are:
+
+* Is it really one idea or is it many?
+* What problem are you solving?
+* Why is what you are suggesting better than what's already there?
+
+
+
+
+### Submitting Pull Request
```

Comment:
Is this WIP?

Reply:
Yes, I'm still working on it and it would take longer time than I expected. I suggest remove this section for now, and merge it to master later together with a updated version of documentation.

Reply:
Works for me!

Reply:
Nice! Check it out here https://github.com/Morhaus/dispatch-proxy/pull/83/commits/09075f307f35b0777789a595764859e47b7d5668

Reply:
It's still missing the section removal.